optaplanner-operator/src/k8s/school-timetabling-solver.yml

```diff
@@ -4,8 +4,17 @@ kind: Solver
 metadata:
   name: school-timetabling
 spec:
-  kafkaCluster: my-cluster
-  kafkaBootstrapServers: my-cluster-kafka-bootstrap.demo.svc.cluster.local:9092
+  amqBroker:
+    host: ex-aao-amqp-0-svc.demo.svc.cluster.local
```

Comment:
The host is a service created by enabling the AMQP acceptor (the quickstart README shows that).